tdrs-backend/gunicorn_start.sh

```diff
@@ -26,7 +26,7 @@ sleep 5
 celery -A tdpservice.settings beat -l info --scheduler django_celery_beat.schedulers:DatabaseScheduler &
 
 # rebuild and repopulate elastic search indexes
-python manage.py search_index --rebuild -f
+# python manage.py search_index --rebuild -f
```

Comment:
Should this be commented out?

Reply:
yeah, this was causing some issues with the database migrations when there was already data in the deployed environment. it's not really needed as elastic will auto-migrate indexed data (safely) on deployment anyway.